liveblog/liveblog
# [LBSD-2799] - Issues with remote advertising

### Purpose

This PR fixes an issue where remote advertising enabled in outputs was rendering in wrong frequencies.

### What has changed

- Refactored the `renderAds` function. 
- Changed from `math.ceil` to `math.floor` for better consistency.

### Steps to test
1. Open the Editor.
2. Go to Ads Manager and create a new ad and add it to a collection.
3. Create a new blog under whose settings, you create an output channel with the created ad collection.
4. Publish a couple of posts.
5. Open the output embed and observe that the ads show as expected.


Resolves: [LBSD-2799](https://sofab.atlassian.net/browse/LBSD-2799)

[LBSD-2799]: https://sofab.atlassian.net/browse/LBSD-2799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
